src/tutorial/packaging.md

```diff
@@ -161,7 +161,7 @@ automatically?
 
 ### Building binary releases on CI
 
-If your tool is open source,
+If your tool is open sourced
```

Comment:
Hm, not sure I'd make this a past participle. My goal was saying being "open source" is an attribute of the code. But this change reads fine, let's keep it!